src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java

```diff
@@ -193,12 +193,12 @@ public void testEnumsAndAnnotations() throws Exception {
         checkConfig.addAttribute("tokens",
             "ANNOTATION_DEF, ENUM_DEF, ANNOTATION_FIELD_DEF, ENUM_CONSTANT_DEF");
         final String[] expected = {
-            "9: " + getCheckMessage(MSG_WRITE_TAG, "@incomplete", "This enum needs more code..."),
-            "13: " + getCheckMessage(MSG_WRITE_TAG, "@incomplete",
+            "18: " + getCheckMessage(MSG_WRITE_TAG, "@incomplete", "This enum needs more code..."),
+            "22: " + getCheckMessage(MSG_WRITE_TAG, "@incomplete",
                 "This enum constant needs more code..."),
-            "19: " + getCheckMessage(MSG_WRITE_TAG, "@incomplete",
+            "28: " + getCheckMessage(MSG_WRITE_TAG, "@incomplete",
                 "This annotation needs more code..."),
-            "23: " + getCheckMessage(MSG_WRITE_TAG, "@incomplete",
+            "32: " + getCheckMessage(MSG_WRITE_TAG, "@incomplete",
                 "This annotation field needs more code..."),
         };
         verify(checkConfig, getPath("InputWriteTag2.java"), expected);
```

Comment:
could you please rename it according to test case

Reply:
Done